Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -733,7 +733,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = 43253H4X22;
 				INFOPLIST_FILE = MockVenmo/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
Same question here on the deployment target changes.

Reply:
I think we also want to revert these demo app deployment targets back to 14. Generally we try to keep our demo app at the same minimum version our SDK supports.

Reply:
Strange I thought I reverted then all but didn't. Should be good now!

Reply:
🤔 I'm not sure that commit got pushed since we are still seeing a diff here.

Reply:
Oops sorry, should be fixed now